_transactions.md

```diff
@@ -267,6 +267,140 @@ url        | The URL of the 3DSecure confirmation request.
+
+To obtain the beneficiary requirements (or validate the `beneficiary` object) when preparing a transaction,
+use the `?validate=true` parameter in the query string of the request to create a transaction.
+This will generate a validation error if any required beneficiary information is missing.
+Otherwise, the transaction will fail at the commit step with a similar error message.
+
+<aside class="notice">
+  Please note that at this moment, even with the <code>validate=true</code> parameter,
+  the validation is only performed if a <code>beneficiary</code> object is passed.
+</aside>
+
+> Example of including the beneficiary information when creating an uncommitted transaction (i.e. quote), alongside the remaining transaction data:
+
+```bash
+curl 'https://api-sandbox.uphold.com/v0/me/cards/<card-id>/transactions' \
+  -H 'authorization: Bearer <bearer-token>' \
+  -H 'content-type: application/json' \
+  -d '{ "beneficiary": { "relationship": "child", "name": "Harper Wilson", "address": { "line1": "Bar", "zipCode": "12345", "city": "Braga", "country": "PT", "state": "PT-03" } }, "purpose": "donations", "denomination": { "amount": "3000", "currency": "USD" }, "destination": "invite-user@gmail.com" }'
+```
+
+> Example of adding the beneficiary information when committing a previously created transaction:
+
+```bash
+curl 'https://api-sandbox.uphold.com/v0/me/cards/<card-id>/transactions/<transaction-id>/commit' \
+  -H 'authorization: Bearer <bearer-token>' \
+  -H 'content-type: application/json' \
+  -d '{ "beneficiary": { "relationship": "child", "name": "Harper Wilson", "address": { "line1": "Bar", "zipCode": "12345", "city": "Braga", "country": "PT", "state": "PT-03" } }, "purpose": "donations" }'
+```
+
+> In both cases, incomplete beneficiary information will be reported in a format similar to this:
+
+```json
+{
+  "code": "validation_failed",
+  "errors": {
+    "beneficiary": {
+      "code": "validation_failed",
+      "errors": {
+        "name": [
+          {
+            "code": "required",
+            "message": "This value is required"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+The beneficiary information can be passed either when creating an uncommitted transaction (i.e. a quote)
+or when [committing it](#step-2-commit-transaction).
+
+> Invalid beneficiary information will be reported like this:
+
+```json
+{
+  "code": "validation_failed",
+  "errors": {
+    "beneficiary": {
+      "code": "validation_failed",
+      "errors": {
+        "name": [
+          {
+            "code": "invalid_beneficiary",
+            "message": "The provided beneficiary is invalid"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+<aside class="notice">
+  For regulatory compliance reasons, the beneficiary name is checked by a sanctions screening process,
+  and is expected to consist entirely of characters in the Latin, Cyrillic, Greek or Georgian alphabets,
```

Comment:
Should we add a link here to the part of our documentation where the actual ranges are shown?

Reply:
Yes, a link is to be added here once that section is created. That's a separate PR that hasn't been created yet.